in_toto/gpg/formats.py

```diff
@@ -69,7 +84,9 @@
 )
 
 
-RSA_PUBKEY_SCHEMA = ssl_schema.Object(
+# Internal temporary rsa public key schema, required for self-referential
```

Comment:
Why is this temporary?  The comment should explain.  It might help a future developer working on this to understand why it's temporary, and why it should later be removed.